### PR TITLE
ADD more markers #START_MONTH# and #END_MONTH#

### PR DESCRIPTION
**Pull request by gonxi**
ADD more markers, #START_MONTH# and #END_MONTH#

### CODE REVIEW

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Remove 'text' word in your additions.

**Comment by gonxi** [comment]
ok!

**Comment by StefanRijnhart** [comment]
How about allowing for a template string passed to strftime instead of adding more and more custom substitutions?

**Comment by lasley** [comment]
I second that idea. Playing off the current format, we could just embed the string into the markers. Maybe something like `#START:%B %Y#`, then just split by `:` & use the internal date format if included. Otherwise fallback to the date format arg in order to provide back compatibility.

**Comment by ovnicraft** [comment]
@StefanRijnhart i start working with `contract` module so you recomendation will have locale issue ?  `datetime` is locale aware but IDK about `strftime`

**Comment by lasley** [comment]
@ovnicraft - By locale, do you mean the ability to put months in as different languages?

File: contract_recurring_invoicing_marker/__openerp__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     'name': 'Markers for contract recurring invoices',
     'version': '8.0.1.0.0',
     'category': 'Contract Management',
-    'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza, '
+    'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza, bisnesmart - Gonzalo Borrás,'
```
Comment (elicoidal) [inline contract_recurring_invoicing_marker/__openerp__.py:9]: usually we put companies in that section as the contributors are in the README

Comment (pedrobaeza) [reply contract_recurring_invoicing_marker/__openerp__.py:9]: Yeah, the exception was me, that my name was my company. But this is no more, as we have now Tecnativa, so the correct is: `Tecnativa, bisnesmart,`

Comment (pedrobaeza) [reply contract_recurring_invoicing_marker/__openerp__.py:9]: Anyway, I'm not sure if this contribution is enough to raise Gonzalo as co-author. @lasley mentioned an interesting rule: if the contribution is about 20-30% of new/refactored code, it deserves to be here. If not, then it should go to contributors section on README.

Comment (lasley) [reply contract_recurring_invoicing_marker/__openerp__.py:9]: Keep in mind that the author key is a limited field, and I'm pretty sure it's not even the full 255 chars of the field.

I'll submit a draft change to our contribution guidelines today on this. I've been testing the rule internally, but quite a few heavy contributors seemed to like it when they saw it. Figure it's time to make the suggestion official now that it's public 😄

Comment (gonxi) [reply contract_recurring_invoicing_marker/__openerp__.py:9]: For me it is not a problem to appear here or in the readme. I only added my name by convention

**Comment by StefanRijnhart** [comment]
@ovnicraft Good point. Maybe use Babel's format_date instead. http://babel.pocoo.org/en/latest/dates.html

**Comment by rafaelbn** [comment]
Hi @gonxi , please could you review travis in order to get it green? 😄

**Comment by pedrobaeza** [comment]
Any news about this. If requested changes are not performed, I will proceed to close it for cleaning this.

**Comment by pedrobaeza** [comment]
Closing as no further answer.

**Pull request closed without merging**